dashboard/modules/metrics/dashboards/data_dashboard_panels.py

```diff
@@ -104,7 +116,7 @@
         ],
     ),
     Panel(
-        id=9,
+        id=10,
```

Comment:
let's not modify the panelId of existing graphs since we've released metrics to the public already. This is considered a backwards-incompatible change. It's possible users may not update their grafana dashboards after they upgrade their ray version.

Instead, just use a new panelId for the new Rows outputted graph